src/elements/pictureChoose/pictureChoose.tsx

```diff
@@ -1,17 +1,37 @@
 import noImg from "images/noimg.png";
+import { ChangeEvent } from "react";
 import styles from "./pictureChoose.module.scss";
 
 interface IPictureChooseProps {
   picture: string;
   className: string;
   buttonClassName: string;
+  onChange: ((value: React.SetStateAction<string>) => void) | ((value: string) => void);
 }
 
-const PictureChoose = (props: IPictureChooseProps) => (
-  <div className={props.className}>
-    <img src={noImg} alt={noImg} className={styles.image} />
-    <input type="file" name="Select image" className={`${props.buttonClassName} ${styles.button}`} />
-  </div>
-);
+const PictureChoose = (props: IPictureChooseProps) => {
+  const changeProcessor = (e: ChangeEvent) => {
+    const reader = new FileReader();
+    const { files } = e.target as HTMLInputElement;
+    reader.onloadend = (event: ProgressEvent<FileReader>) => {
+      if (event?.target?.result != null) props.onChange(event?.target?.result as string);
+    };
+    if (files != null) reader.readAsDataURL(files[0]);
+  };
+
+  const pictureSrc = props.picture === "" ? noImg : props.picture;
+
+  return (
+    <div className={props.className}>
```

Comment:
send instructions in skype message

Reply:
instructions followed. Photo is saved in localStorage